docs/src/examples/components/Segment/Variations/SegmentExampleInverted.shorthand.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react'
+import { Segment, Text } from '@stardust-ui/react'
+
+const SegmentExampleInvertedShorthand = () => (
+  <div>
+    <Segment content="Colored segment." variables={{ color: 'purple' }} />
```

Comment:
Color in this case is a primary and common concern for the Segment.  It should then be available as a prop:

```jsx
<Segment color='purple' />
```

Reply:
this would be new API, we don't have `color` as prop in the API of any component...
@levithomason @kuzhelov

Reply:
going to resolve for now we can reopen if we realize we didn't follow the right approach

Reply:
I would leave it as a variable until we figure out if we want to expose the color as a prop to different components.

Reply:
added `color` prop per discussion, @levithomason can you review again?